torch/_inductor/graph.py

```diff
@@ -876,7 +887,12 @@ def get_attr(self, target, args, kwargs):
         if isinstance(value, torch.fx.GraphModule):
             return ir.Subgraph(name=target, graph_module=value)
 
-        if (
+        elif isinstance(value, torch._C.ScriptObject):
+            self.constants[target] = value
+            # self.constant_reprs[target] = ??
```

Comment:
what is this ??

Reply:
I'm not sure? I saw it when we're [creating tensor constants](https://fburl.com/code/sww1tsmp) and was wondering if this is needed here too

Reply:
Oh I realized, it's used as a comment for when we declare the constants globally ([code pointer](https://www.internalfb.com/code/fbsource/[c1a73c1091ecd3250d74b396275058ea412f062c]/fbcode/caffe2/torch/_inductor/codegen/wrapper.py?lines=464-466%2C486-487)):
```
_tensor_constant0 = None  # constant_reprs["_tensor_constant0"] 
```